showcase/app/controllers/page-components/stepper/list.ts

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { deepTracked } from 'ember-deep-tracked';
+
+import type { PageComponentsStepperListModel } from 'showcase/routes/page-components/stepper/list';
+import { clone } from 'showcase/utils/clone';
```

Comment:
[question/thought] why define a utility in a file vs defining it as local function just in this file, like it was before (in the router)? the existing comment says `// basic function that clones an array of objects (not deep)` so it's very specific, not sure moving it in a global level with such generic name is correct

Reply:
a separate ticket then, where we make the changes everywhere?